Source/Core/Core/IOS/WFS/WFSI.cpp

```diff
@@ -215,8 +216,13 @@ IPCCommandResult WFSI::IOCtl(const IOCtlRequest& request)
     break;
 
   case IOCTL_WFSI_INIT:
-    // Nothing to do.
     INFO_LOG(IOS, "IOCTL_WFSI_INIT");
+    if (GetIOS()->GetES()->GetTitleId(&m_title_id) < 0)
```

Comment:
Nitpick: shouldn't you be checking against `IPC_SUCCESS`, just to be consistent?